package.json

```diff
@@ -22,6 +22,7 @@
   "devDependencies": {
     "concurrently": "^4.0.1",
     "cypress": "6.2.0",
+    "cypress-iframe": "^1.0.1",
```

Comment:
i'll share handler of iframes when you get closer to those test
but for now please remove this package

Reply:
done